shortest_superseq_problem/BUILD.bazel

```diff
@@ -0,0 +1,23 @@
+cc_library(
+    name = "shortest_superseq",
+    srcs = ["shortest_superseq.cpp"],
+    hdrs = ["shortest_superseq.h"],
+    deps = ["@abseil-cpp//absl/container:flat_hash_set"],
+    copts = ["/std:c++20"],
+)
+
+cc_test(
+    name = "shortest_superseq_test",
+    srcs = ["shortest_superseq_test.cpp"],
+    deps = [":shortest_superseq","@googletest//:gtest_main"],
```

Comment:
super-nit:
use 'buildifier' to format the BUILD file, same as you use clang-format to format the C++ files. 
With proper plugins (either the editor or git  pre-commit) you can have this done automatically.